workflow-templates/php-library-security.yml

```diff
@@ -0,0 +1,17 @@
+name: Security analysis
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
May be better to use `$default-branch` keyword for any instances of hardcoding branch names in templates.